clusterloader2/pkg/measurement/util/wait_for_conditions.go

```diff
@@ -145,3 +146,7 @@ func countObjectsMatchingConditions(objectConditions [][]metav1.Condition, succe
 func conditionToKey(c metav1.Condition) string {
 	return fmt.Sprintf("%s=%s", c.Type, c.Status)
 }
+
+func objectToDebugName(o ObjectSimplification) string {
```

Comment:
Nit: maybe rename to `String()` receiver method or `Name()` receiver method (as we do not print everything)?

Reply:
done